indra/llmath/llvolume.h

```diff
@@ -1113,9 +1120,9 @@ class LLVolume : public LLRefCount
 
 public:
     virtual void setMeshAssetLoaded(bool loaded);
-    virtual bool isMeshAssetLoaded();
+    virtual bool isMeshAssetLoaded() const;
     virtual void setMeshAssetUnavaliable(bool unavaliable);
-    virtual bool isMeshAssetUnavaliable();
+    virtual bool isMeshAssetUnavaliable() const;
```

Comment:
it looks like these methods are not overridden in any subclasses (I found LLModel and LLRiggedVolume)?  so this won't break them.  maybe in the future we should make these non-virtual?

Reply:
Yea, I'm not sure why these are virtual.